### DISCUSSION TITLE
New file detection for language qualifier: pikchr

### DISCUSSION THREAD

**Discussion by MichaelHinrichs** [community/community#123517] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I would like the [pikchr](https://pikchr.org/home/doc/trunk/homepage.md) markup language to be added to the searchable languages.

Here is a repo that uses it.
https://github.com/gopikchr/gopikchr/tree/main

It uses the file extension ".pikchr".

**Comment by look** [comment, marked as answer]
Hi @MichaelHinrichs, this is handled by the [Linguist](https://github.com/github-linguist/linguist) library. You can [request a new language by filing an issue](https://github.com/github-linguist/linguist/issues/new?template=new_language.md).

**Reply by MichaelHinrichs** [reply]
Thank you. I would have found that, if it were owned by [GitHub](https://github.com/github) instead of a completely separate organization that i didn't know about.
